src/components/MovieFilter/MovieFilter.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { MoviesList } from '../MoviesList';
+
+export class MovieFilter extends Component {
+  state = {
+    query: '',
+  };
+
+  handleChange = (event) => {
+    this.setState({
+      query: event.target.value,
```

Comment:
how about converting it in lowercase here?

Reply:
done, but now I just can't type a capital letter in the search bar

Reply:
I see, you're right, forgot about this thing :)